ksql-rest-app/src/test/java/io/confluent/ksql/rest/server/computation/CommandStoreTest.java

```diff
@@ -120,12 +123,13 @@ public void shouldHaveAllCreateCommandsInOrder() {
   }
 
   @Test
-  public void shouldFailEnqueueIfCommandWithSameIdRegistered() {
+  public void shouldFailEnqueueIfCommandWithSameIdRegistered() throws InterruptedException, ExecutionException {
```

Comment:
```suggestion
  public void shouldFailEnqueueIfCommandWithSameIdRegistered() throws Exception {
```

No point having long lists of exception types being thrown in tests. They change over time...